_posts/2022-03-08-PC-Engines-Firmware-v4.16.0.1.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title:  "PC Engines apu coreboot Open Source Firmware v4.16.0.1"
+date:   2022-03-08
+categories: Firmware
+---
+
+# PC Engines apu coreboot Open Source Firmware v4.16.0.1
+
+## Key changes
+
+Mainline:
+
+### Changed
+- **Rebased** with official coreboot repository **commit** 36425312ee
```

Comment:
@IgorBagnucki this commit is the same as in the rel 4.15.0.3

Reply:
Thanks for catching that. Fixed